boards/nxp/lpcxpresso54114/doc/index.rst

```diff
@@ -47,7 +47,7 @@ For more information about the LPC54114 SoC and LPCXPRESSO54114 board:
 Supported Features
 ==================
 
-The lpcxpresso54114 board configuration supports the following hardware
+The ``lpcxpresso54114`` board target supports the following hardware
```

Comment:
here

Reply:
Thanks, I've added this to the opt-out list.